src/main/java/org/jabref/gui/externalfiles/DroppedFileHandler.java

```diff
@@ -351,18 +350,19 @@ private boolean showLinkMoveCopyRenameDialog(String linkFileName, ExternalFileTy
         moveRadioButton.setText(Localization.lang("Move file to file directory"));
         renameCheckBox.setText(Localization.lang("Rename file to").concat(": "));
 
-        LayoutFormatterPreferences layoutPrefs = Globals.prefs
-                .getLayoutFormatterPreferences(Globals.journalAbbreviationLoader);
+        // FIXME: Remove unused code later S.G.
```

Comment:
Please remove the commented-out code here.

Reply:
Removed; will push in a short while.